lib/jazzy/sourcekitten.rb

```diff
@@ -340,6 +342,18 @@ def self.autolink(docs, data)
       end
     end
 
+    def self.reject_objc_enum_typedefs(docs)
```

Comment:
I've only tested this on Realm, which doesn't have any structs in its public API. So yes, this would likely have to be expanded for structs.

Repeatedly, it'd be great to add another, well-documented ObjC library. Thoughts?